bulletroboy/operator/operator_hardware.py

```diff
@@ -0,0 +1,89 @@
+from rclpy.callback_groups import ReentrantCallbackGroup
+
+from geometry_msgs.msg import PoseStamped
+from roboy_middleware_msgs.msg import MotorCommand
+from .operator import Operator, Link
+from ..utils.utils import Topics, Services
+
+import time
+
+PULL_MOTOR = 16
+
+class OperatorHW(Operator):
+	"""This class represents the "real" Operator and communicates directly with the animus_server.
+
+	"""
+	def __init__(self):
+		"""
+		Args:
+			-
+
+		"""
+		super().__init__()
+
+	def start_node(self):
+		self.pose_subscriber = self.create_subscription(PoseStamped, Topics.VR_HEADSET_POSES, self.vr_pose_listener, 1, callback_group=ReentrantCallbackGroup())
+		self.motor_command_publisher = self.create_publisher(MotorCommand, Topics.MOTOR_COMMAND, 1)
+		input("\nStand in initial position and press enter...")
+		self.pull()
+		self.start_publishing()
+
+	def init_links(self):
+		"""Initiates the links in operator class.
+
+		Args:
+			-
+
+		Returns:
+			-
+
+		"""
+		self.links = []
+		for i, key in enumerate(self.link_map):
+			human_name = self.link_map[key]
+			roboy_name = key
+			dims = self.get_parameter("operator_link_dimentions." + human_name).get_parameter_value().double_array_value
+			self.links.append(Link(i, human_name, roboy_name, dims))
+
+	def pull(self):
+		"""Pulls operator to simulate connection to the roboy.
+
+		Args:
+			-
+
+		Returns:
+			-
+			
+		"""
+		msg = MotorCommand()
+
+		msg.legacy = False
+		msg.motor = [PULL_MOTOR]
+		msg.setpoint = [800.0]
```

Comment:
hardcoded values everywhere in the code

Reply:
see https://github.com/Roboy/bulletroboy/pull/25#issuecomment-760128816